.github/workflows/test-all-packages.yml

```diff
@@ -282,6 +326,43 @@ jobs:
 
 ##############
 # Long-running tests are executed individually.
+  test-solo:
```

Comment:
Good and correct, but this reminds me how nice it would be if we could simplify or factor out some of these cache-management steps, which I don't really understand very well, and make me nervous every time I need to touch something in here. Not that I have any idea how to approach such a thing, nor if there's any real improvement to be had.

Reply:
> if there's any real improvement to be had.

Short of Github Actions sprouting features to abstract the workflows, we could also use a script to generate the files.  Not sure that's better, though.